systemtest/elasticsearch.go

```diff
@@ -95,6 +95,7 @@ func CleanupElasticsearch(t testing.TB) {
 		esapi.IndicesDeleteRequest{Index: []string{prefix}},
 		esapi.IngestDeletePipelineRequest{PipelineID: prefix},
 		esapi.IndicesDeleteTemplateRequest{Name: prefix},
+		esapi.IndicesRefreshRequest{},
```

Comment:
This should be called _after_ the other requests are done and _before_ the ILM request is done. In https://github.com/elastic/apm-server/pull/4434/files#diff-1b744e35d185e772edc9c58ab2f27cd51c987e60bbc821d71781b06769d7df34R112 however the requests are run in parallel, so I believe this should be after these requests.

Reply:
Indeed, thanks :man_facepalming: